client/my-sites/plans/jetpack-plans/jetpack-complete-page/components/cta-buttons/style.scss

```diff
@@ -0,0 +1,40 @@
+.jetpack-complete-page__cta-buttons {
```

Comment:
According to the designs, our buttons have the following specs:

- `border-radius` of 4px
- `font-weight` of 600

We need new rules for these inside the `__cta-buttons` scope, probably targeted at `.button` or `.form-button`.